source/server/server.cc

```diff
@@ -80,7 +81,7 @@ InstanceImpl::~InstanceImpl() {
 
   // Stop logging to file before all the AccessLogManager and its dependencies are
   // destructed to avoid crashing at shutdown.
-  Logger::Registry::getSink()->logToStdErr();
+  file_logger_.reset(nullptr);
```

Comment:
nullptr is the default value; envoy convention is to omit it as an arg at the callsite.